src/public/orders/methods.ts

```diff
@@ -25,4 +25,4 @@ export type ShipmentCreated = (transaction: Transaction, shipment: SalesOrderShi
  * Called when an order has been imported into a marketplace.
  */
 export type AcknowledgeOrders = (transaction: Transaction, notifications: SalesOrderNotification[])
-=> AcknowledgedSalesOrder[] | Promise<AcknowledgedSalesOrder[]>;
\ No newline at end of file
+=> AcknowledgedSalesOrder[] | Promise<AcknowledgedSalesOrder[]>;
```

Comment:
So good

Reply:
Doing the lords work, one new line at a time.